### ISSUE TITLE
Matching Order in LxmlMicrodataExtractor._extract_property_value 

### ISSUE THREAD

**Issue by kelvinso** [scrapinghub/extruct#160]
I noticed that the matching order of _extract_property_value seems to be inconsistent with https://www.w3.org/TR/microdata/#values. In this doc, it mentions that the 2nd matching case is  "If the element has a content attribute". However, in LxmlMicrodataExtractor._extract_property_value, it is 2nd to the last in the matching order. 

Should this case
```
 elif node.get("content"):
            return node.get("content")
```
in w3cmicrodata.py be moved before resolving for meta tag at line 186?

Thanks a lot!
Kelvin

**Comment by Gallaecio** [comment]
Yeah, it looks like the changes they’ve made to the specification since [2013](https://www.w3.org/TR/2017/WD-microdata-20170504/) (that code is from 2014) include allowing `content` on any node, which back in 2013 was non-standard yet supported by extruct.
